pkgs/by-name/ge/gerbolyze/package.nix

```
@@ -0,0 +1,95 @@
+{ lib
+, stdenv
+, python3Packages
+, fetchFromGitHub
+, resvg
+}:
+
+let
+  version = "3.1.7";
+  src = fetchFromGitHub {
+    owner = "jaseg";
+    repo = "gerbolyze";
+    rev = "v${version}";
+    hash = "sha256-0iTelSlUJUafclRowwsUAoO44nc/AXaOKXnZKfKOIaE=";
+    fetchSubmodules = true;
+  };
+
+  svg-flatten = stdenv.mkDerivation rec {
+    inherit version src;
+    pname = "svg-flatten";
+
+    sourceRoot = "source/svg-flatten";
+
+    postPatch = ''
+      substituteInPlace Makefile \
+        --replace '$(INSTALL) $(BUILDDIR)/$(BINARY) $(PREFIX)/bin' \
+        '$(INSTALL) $(BUILDDIR)/$(BINARY) $(PREFIX)/bin/svg-flatten'
+    '';
+
+    installPhase = ''
+      runHook preInstall
+      mkdir -p $out/bin
+      PREFIX=$out make install
+      runHook postInstall
+    '';
+
+    meta = with lib; {
+      description = "svg-flatten SVG downconverter";
+      homepage = "https://github.com/jaseg/gerbolyze";
+      license = with licenses; [ agpl3 ];
+      maintainers = with maintainers; [ wulfsta ];
+      mainProgram = "svg-flatten";
+    };
+  };
+in python3Packages.buildPythonApplication rec {
+  inherit version src svg-flatten;
```

the environ variable `$svg-flatten` isn't used.

```suggestion
  inherit version src;
```